### DISCUSSION TITLE
Custom models

### DISCUSSION THREAD

**Discussion by ghost** [Minestom/Minestom#799] [Help] [answered]
Is there are any ability to create custom models? For example, for items or entities (or even blocks, if possible)?

**Comment by DevBlokJochem** [comment, marked as answer]
Yes, but you cant do that with minestom. You need a custom texturepack for the custom models. The only thing you have todo in minestom is setting the nbt data.

**Reply by ghost** [reply]
okay, got it. what do I need to set in nbt?

**Reply by DevBlokJochem** [reply]
The name. If you have your custom model, you can give some values. For example: you give the item model the nbt: "test_model", and you will give the item in your minestom project also the nbt: "test_model", the resourcepack will work. You can watch bukkit/spigot tutorials if you dont know how todo it. Its the same way.

**Reply by ghost** [reply]
Thank you
